src/models/subscription.ts

```diff
@@ -153,6 +153,22 @@ export class Subscription extends Model {
 		});
 	}
 
+	static async findForRepoNameAndOwner(repoName: string, repoOwner: string, jiraHost: string): Promise<Subscription | null> {
```

Comment:
tests?

Reply:
Test case added

Reply:
Btw why is it `sequelize?` Could it be undefined?